src/movies.js

```diff
@@ -2,14 +2,111 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+function orderByYear(movies) {
+  const newArray = [...movies];
+  newArray.sort(function(movA, movB) {
+    if (movA.year < movB.year) {
+      return -1;
+    }
+    if (movA.year > movB.year) {
+      return 1;
+    }
+    if (movA.year === movB.year) {
+      if (movA.title < movB.title) {
+        return -1;
+      }
+      if (movA.title > movB.title) {
+        return 1;
+      }
+      if (movA.title === movB.title) {
+        return 0;
+      }
+      return 0;
+    }
+  });
+  return newArray;
+}
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct
 
+function howManyMovies(movies) {
+  const dramaMovies = movies.filter(function(movie) {
+    if (
+      movie.genre.includes("Drama") &&
+      movie.director === "Steven Spielberg"
+    ) {
+      return true;
+    }
+  });
+  return dramaMovies.length;
+}
+
 // Iteration 3: Alphabetic Order - Order by title and print the first 20 titles
 
+function orderAlphabetically(movies) {
+  const newArray = [...movies];
+  const alfaOrder = newArray
+    .sort(function(movA, movB) {
+      if (movA.title < movB.title) {
+        return -1;
+      }
+      if (movA.title > movB.title) {
+        return 1;
+      }
+      if (movA.title === movB.title) {
+        return 0;
+      }
+      return 0;
+    })
+    .map(function(movie) {
```

Comment:
Good job on joining methods together, however, if they have multiple lines, we would recommend to separate them for better readability.